package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@balena/es-version",
+  "version": "0.0.0",
+  "description": "This module provides the ability to specify the default es version to include for balena modules.",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
```

Comment:
NIT: Append:
```
  "files": [
    "build/"
  ],
```

Reply:
Why?

Reply:
Just to reduce the number of files in the published package.
PS: Npm will still include LICENSE package.json CHANGELOG.md README.md, so it's just src .gitignore tsconfig (and husky & lint-staged configs if they were in a separate files) that would be omitted.